scripts/internal/replay_events_csv.go

```
@@ -0,0 +1,277 @@
+	body, err := json.Marshal(event)
+	if err != nil {
+		return fmt.Errorf("marshal: %w", err)
+	}
+
+	var lastErr error
+	for attempt := 0; attempt <= replayMaxRetries; attempt++ {
+		if attempt > 0 {
+			time.Sleep(replayInitialBackoff * time.Duration(attempt))
+		}
+
+		req, err := http.NewRequest(http.MethodPost, endpoint, bytes.NewReader(body))
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)